app/src/main/AndroidManifest.xml

```diff
@@ -47,17 +52,15 @@
             android:name=".view.activity.MainActivity"
             android:hardwareAccelerated="true"
             android:label="@string/app_name"
-            android:theme="@style/AppTheme.NoActionBar">
-        </activity>
+            android:theme="@style/AppTheme.NoActionBar"></activity>
 
-        <meta-data
-            android:name="io.fabric.ApiKey"
-            android:value="${fabric_api_key}"/>
+        <!-- <meta-data -->
+        <!-- android:name="io.fabric.ApiKey" -->
+        <!-- android:value="${fabric_api_key}"/> -->
 
         <meta-data
             android:name="com.google.android.maps.v2.API_KEY"
-            android:value="${google_map_api_key}" />
-
+            android:value="AIzaSyAnCO9njYPfNfdGKNEbmt_UJ3_lMDYTAb8" />
```

Comment:
never commit your key...

Reply:
i didn't really change anything on that line, that code is not mine, it came with the code when i cloned the project.

Reply:
I just remembered that @Sparow199 had problem with using parameter for the key in the manieft i think that's why the key is commited directly